jsapp/js/components/submissions/submissionModal.es6

```diff
@@ -169,12 +170,21 @@ class SubmissionModal extends React.Component {
       isEditLoading: true,
       isEditingDuplicate: true,
     });
-    enketoHandler.editSubmission(this.props.asset.uid, this.state.sid).then(
+    enketoHandler.openSubmission(this.props.asset.uid, this.state.sid, 'edit').then(
```

Comment:
Please make `'edit'` and `'view'` a constant defined in `enketoHandler`. I wanted to describe here our way of defining enum-like objects, but ended up improving our ways and there is new PR: kpi#3321. I don't know how much more work you have here, but if the `createEnum` PR is merged first, please use it :)